tests/multisigNFTs_test.go

```diff
@@ -0,0 +1,192 @@
+		{"multiSig", false, false, "[case-2.0] Create MultiSig Tx for NFTs [Accept-token-ownership] - [internalTx3x_acceptTokenOwnership_err3] - wait-15-seconds.", "multisig-acc-nft-4", "100000000cin", 0, MultisigInfo{"create-internal-tx", "", "", 0, nil, "nft-grp-addr-4", 2, internalTx3x_acceptTokenOwnership_err3}, "MEMO : xxxxx", nil}, // nx "sequence":
+		{"multiSig", false, false, "[case-2.0] Sign MultiSig Tx for NFTs [Accept-token-ownership] - submit which signed by multisig-acc-nft-4 - wait-15-seconds.", "multisig-acc-nft-4", "100000000cin", 0, MultisigInfo{"multiSig-sign-tx", "", "", 0, nil, "nft-grp-addr-4", 0, internalTx3x_acceptTokenOwnership1}, "MEMO : Sign MultiSig Tx", nil},
+		{"multiSig", false, false, "[case-2.0] Sign MultiSig Tx for NFTs [Accept-token-ownership] - submit which signed by multisig-acc-nft-3 - wait-15-seconds.", "multisig-acc-nft-3", "100000000cin", 0, MultisigInfo{"multiSig-sign-tx", "", "", 0, nil, "nft-grp-addr-4", 0, internalTx3x_acceptTokenOwnership1}, "MEMO : Sign MultiSig Tx", nil},
+
+		//Multisig Process : Multisig-transfer-ownership
+		{"multiSig", true, true, "[case-2.0] Transfer MultiSig Owner - Error, due to Group address invalid. - wait-15-seconds.																															 ", "multisig-acc-nft-1", "100000000cin", 0, MultisigInfo{"transfer-ownership", "multisig-acc-nft-1", "multisig-acc-nft-4", 0, nil, "nft-grp-addr-5", 0, nil}, "MEMO : Transfer MultiSig Owner.", nil},
+		{"multiSig", true, true, "[case-2.0] Transfer MultiSig Owner - Error, due to Owner of group address invalid. - wait-15-seconds.																											 ", "multisig-acc-nft-2", "100000000cin", 0, MultisigInfo{"transfer-ownership", "multisig-acc-nft-2", "multisig-acc-nft-4", 0, nil, "nft-grp-addr-3", 0, nil}, "MEMO : Transfer MultiSig Owner.", nil},
+		{"multiSig", true, true, "[case-2.0] Transfer MultiSig Owner - Error, due to without KYC - wait-15-seconds.																																					 ", "multisig-nokyc", "100000000cin", 0, MultisigInfo{"transfer-ownership", "multisig-acc-nft-1", "multisig-nokyc", 0, nil, "nft-grp-addr-5", 0, nil}, "MEMO : Transfer MultiSig Owner.", nil},
+		{"multiSig", false, false, "[case-2.0] Transfer MultiSig Owner - [from multisig-acc-nft-1 to multisig-acc-nft-4] Happy Path - commit. - wait-15-seconds.																		 ", "multisig-acc-nft-1", "100000000cin", 0, MultisigInfo{"transfer-ownership", "multisig-acc-nft-1", "multisig-acc-nft-4", 0, nil, "nft-grp-addr-3", 0, nil}, "MEMO : Transfer MultiSig Owner.", nil},
+		{"multiSig", true, true, "[case-2.0] Re-transfer MultiSig Owner - Error, due to Owner of group address invalid [MultiSig-account already been transfer to others]. - wait-15-seconds.", "multisig-acc-nft-1", "100000000cin", 0, MultisigInfo{"transfer-ownership", "multisig-acc-nft-1", "multisig-acc-nft-4", 0, nil, "nft-grp-addr-3", 0, nil}, "MEMO : Transfer MultiSig Owner.", nil},
+
+		// signer without through KYC
```

Comment:
I think this case is redundant. We have tested this in multisig_test.go

Reply:
Mostafa,

You are correct. Will remove the above once come to merging with other test-modules.

Reply:
Mostafa,

I take note your comment here.